### ISSUE TITLE
Temporal grid overlay

### ISSUE THREAD

**Issue by mattijn** [felixpalmer/procedural-gl-js#3]
This is great! Next to markers and features as overlay, could this application also render temporal grid overlay to visualise for example flood events?

**Comment by felixpalmer** [comment]
I'm not sure I understand exactly what you mean. Are you asking if you could animate the overlays? There is a API function [updateOverlay](https://felixpalmer.github.io/procedural-gl-js/docs/module-Features.html#.updateOverlay) which lets you update an existing overlay, which you could use to drive an animation.

**Comment by mattijn** [comment]
Thanks for your response! From my understanding, I can create a 3D map experience using raster imagery tiles. But if I have another source of raster imagery tiles, can I render these on top of the map? Based on the link you provided, I understand it is possible for a GeoJSON FeatureCollection, but I was referring to another grid.

**Comment by felixpalmer** [comment]
Ah OK, I take it that it that the raster tiles would be transparent? This should be doable, I'll keep it in mind for a future version. If you have any good freely available datasets that are already in the XYZ tile format that would be helpful!

**Comment by olwal** [comment]
I agree with @mattijn that animated raster tiles would be interesting, and I would add that it would be useful to not only do 2D tiles, but also support depth map tiles, similarly to how you are using tile maps for the elevation.

**Comment by olwal** [comment]
I am using repeated addOverlay calls to show temporal data, since updateOverlay has limitations (can only update the coordinates, # of features need to match exactly, etc). 

I think that it would add a lot of power with a more flexible updateOverlay method, which could allow high-performance animation of colors, transparency and geometry. It would also be useful to support updates where the numbers of features are not the same in each update (missing data, etc.) -- as you can see in the attached example, the sensor data vary quite a bit. 

https://user-images.githubusercontent.com/517681/103614464-86214b80-4edd-11eb-901b-9f5b5ab02716.mov

**Comment by felixpalmer** [comment]
@olwal I've created a new issue to track the `updateOverlay` enhancement: https://github.com/felixpalmer/procedural-gl-js/issues/17 so this issue can be purely for the original suggestion of @mattijn 

Really great work on the air quality visualization by the way! Super excited to see what this develops into
